core/src/util.jl

```diff
@@ -383,17 +378,15 @@ function get_level(
     storage::Union{AbstractArray, Number} = 0,
 )::Union{Real, Nothing}
     (; basin, level_boundary) = p
-    hasindex, i = id_index(basin.node_id, node_id)
-    current_level = get_tmp(basin.current_level, storage)
-    return if hasindex
+    return if node_id.type == NodeType.Basin
```

Comment:
```suggestion
   if node_id.type == NodeType.Basin
```

I am fine with having "return" in general if @Huite insists, but here it really doesn't make sense :P